src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Application from './view/application/Application';
+import './style.css';
```

Comment:
Library imports, absolute imports and relative imports, order by this rule.

Reply:
done